app/models/pos.py

```diff
@@ -0,0 +1,129 @@
+import spacy
+import re
+
+_nlp = spacy.load("en_core_web_sm")
```

Comment:
May need to download model first. Find out a way to do it via a Python function.